src/style.css

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
+  box-sizing: border-box;
+}
+
+.header__container {
+  margin: 0 auto;
+  padding: 0 50px;
+  display: flex;
+  max-width: 1200px;
+  background-color: #ffff;
+  justify-content: space-between;
+  align-items: center;
+  gap: 60px;
+  flex-wrap: wrap;
+}
+
+.logo {
+  max-width: 40px;
+  width: 100%;
+}
+
+.nav__list {
+  display: flex;
+  flex-wrap: wrap;
+  font-family: ' Roboto', sans-serif;
```

Comment:
There is an extra space before 'Roboto' in the `font-family` declaration. It should be `font-family: 'Roboto', sans-serif;` without the leading space.